vm/src/stdlib/symtable.rs

```diff
@@ -48,57 +141,39 @@ mod symtable {
 
     #[pyclass]
     impl PySymbolTable {
-        #[pymethod]
-        fn get_name(&self) -> String {
+        #[pygetset]
+        fn name(&self) -> String {
             self.symtable.name.clone()
         }
 
-        #[pymethod]
-        fn get_type(&self) -> String {
+        #[pygetset(name = "type")]
+        fn typ(&self) -> String {
             self.symtable.typ.to_string()
         }
 
-        #[pymethod]
-        const fn get_lineno(&self) -> u32 {
+        #[pygetset]
+        const fn lineno(&self) -> u32 {
             self.symtable.line_number
         }
 
-        #[pymethod]
-        const fn is_nested(&self) -> bool {
-            self.symtable.is_nested
-        }
-
-        #[pymethod]
-        const fn is_optimized(&self) -> bool {
-            matches!(
-                self.symtable.typ,
-                CompilerScope::Function | CompilerScope::AsyncFunction
-            )
```

Comment:
let's not delete these methods but change to getset, just like get_name and get_lineno.
Unlike other a little bit more complex cases, the deletion of these 2 methods cause unnecessary regression.